Add cloud-init foo for configuring Crowdstrike Falcon sensor

jsf9k opened the pull request:
## 🗣 Description ##

This pull request adds some cloud-init foo for configuring the Crowdstrike Falcon sensor.

## 💭 Motivation and context ##

The Crowdstrike Falcon sensor will not function as expected unless properly configured.

## 🧪 Testing ##

All automated tests pass.  I deployed a staging AMI with these changes and verified that it configured the Crowdstrike Falcon sensor as expected.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
- [x] All new and existing tests pass.

jsf9k committed:
Add cloud-init foo for configuring Crowdstrike Falcon sensor

dav3r reviewed (approved):
Very nice, no notes from me. 🦅

mcdonnnj commented on cloud-init/configure-falcon-sensor.py:
```diff
@@ -0,0 +1,126 @@
+        RoleArn=SSM_READ_ROLE_ARN,
+        RoleSessionName="set_crowdstrike_falcon_sensor_customer_id",
+    )
+    newsession_id = stsresponse["Credentials"]["AccessKeyId"]
+    newsession_key = stsresponse["Credentials"]["SecretAccessKey"]
+    newsession_token = stsresponse["Credentials"]["SessionToken"]
```
I'm assuming these are the correct type. Since the dictionary is typed to `dict[str, Any]` we should specify what type we expect these variables to hold.

```suggestion
    newsession_id: str = stsresponse["Credentials"]["AccessKeyId"]
    newsession_key: str = stsresponse["Credentials"]["SecretAccessKey"]
    newsession_token: str = stsresponse["Credentials"]["SessionToken"]
```

mcdonnnj reviewed (approved):
LGTM ✔ One small suggestion for your consideration.

jsf9k committed:
Add types to several variables

Since the dictionary is typed to dict[str, Any] we should specify what type we expect these variables to hold.

Co-authored-by: Nick <50747025+mcdonnnj@users.noreply.github.com>
